operator/Makefile

```diff
@@ -428,6 +434,19 @@ bundle: yq manifests kustomize operator-sdk ## Generate bundle manifests and met
        bundle/manifests/rhacs-operator.clusterserviceversion.yaml
 	$(OPERATOR_SDK) bundle validate ./bundle --select-optional suite=operatorframework
 
+.PHONY: operator-version-for-bundle
+operator-version-for-bundle:
+	@set -euo pipefail ;\
+	$(ACTIVATE_PYTHON) >/dev/null; \
+	first_version=3.62.0 `# 3.62.0 is the first operator version ever released` ;\
+	candidate_version=$$(./bundle_helpers/patch-csv.py \
+		--use-version $(VERSION) \
+		--first-version $${first_version} \
+		--operator-image $(IMG) \
+		--echo-replaced-version-only \
```

Comment:
Why compute the replaced, i.e. previous version of the operator to return it as `operator-version-for-bundle`? I think bundle and controller versions should be 1:1. Besides, I think this target is redundant because `make -C operator tag` should provide what we need.

Reply:
As I said above, the `tag` target is doing somethign different. I used the existing `make bundle-post-process` as a blueprint.